library/src/main/java/com/bumptech/glide/manager/ConnectivityStrategy.java

```diff
@@ -0,0 +1,10 @@
+package com.bumptech.glide.manager;
+
+/** A strategy that to perceive connectivity changes. */
+public interface ConnectivityStrategy {
```

Comment:
Can you use the existing ConnectivityMonitor interface instead of creating a new one?

Reply:
why do that? I'm a little confused.  in my eyes, strategy is mean the  way of check network ,and ConnectivityMoniter only care when check.